### PR TITLE
Corneliu Rusu (HTP): Firefox Privacy Notice update

### PR DESCRIPTION
**Pull request by CorneliuRusu**

### CODE REVIEW

File: it/firefox_privacy_notice.md [hunk 1]
```diff
@@ -26,7 +26,12 @@ Le ricerche in Firefox possono essere eseguite da diversi punti, come la barra d
...
 
-* __Query di ricerca__: se viene selezionato un provider che supporta i suggerimenti di ricerca, per impostazione predefinita, Firefox invia le richieste al provider di servizi di ricerca per aiutare l'utente a scoprire frasi comuni usate nelle ricerche da altri e migliorare l'esperienza di ricerca. [Ulteriori informazioni](https://support.mozilla.org/kb/search-suggestions-firefox) che includono come disattivare questa funzione. Se viene abilitato Firefox Suggest, noi e i nostri partner potremmo anche ricevere le query di ricerca dell'utente. [Ulteriori informazioni di seguito](#searches). 
+* __Query di ricerca__: se viene selezionato un provider che supporta i suggerimenti di ricerca, per impostazione predefinita, Firefox invia le richieste al provider di servizi di ricerca per aiutare l'utente a scoprire frasi comuni usate nelle ricerche da altri e migliorare l'esperienza di ricerca. [Ulteriori informazioni](https://support.mozilla.org/kb/search-suggestions-firefox) che includono come disattivare questa funzione. Se viene abilitato I link ai nostri fornitori di servizi di ricerca predefiniti sono:
```
Comment (flodolo) [inline it/firefox_privacy_notice.md:29]: This translation is wrong, looks like nobody reviewed the translated text? Please talk with the translator, this is not acceptable for a legal document.

There's a half translation from the previous text (`Se viene abilitato`).

File: it/firefox_privacy_notice.md [hunk 2]
```diff
@@ -26,7 +26,12 @@ Le ricerche in Firefox possono essere eseguite da diversi punti, come la barra d
...
+* __Query di ricerca__: se viene selezionato un provider che supporta i suggerimenti di ricerca, per impostazione predefinita, Firefox invia le richieste al provider di servizi di ricerca per aiutare l'utente a scoprire frasi comuni usate nelle ricerche da altri e migliorare l'esperienza di ricerca. [Ulteriori informazioni](https://support.mozilla.org/kb/search-suggestions-firefox) che includono come disattivare questa funzione. Se viene abilitato I link ai nostri fornitori di servizi di ricerca predefiniti sono:
+
+  * [Google](https://policies.google.com/privacy)
+  * [Microsoft (Bing)](https://privacy.microsoft.com/privacystatement)
+
+* Se viene abilitat l'opzione "Improve the Firefox Suggest Experience" (Migliora l'esperienza di Firefox Suggest), noi e i nostri partner potremmo anche ricevere le query di ricerca dell'utente. [Ulteriori informazioni di seguito](#searches).
```
Comment (flodolo) [inline it/firefox_privacy_notice.md:34]: And there's a typo here (abilitat**a**)

File: it/firefox_privacy_notice.md [hunk 3]
```diff
@@ -81,7 +86,7 @@ Leggere la documentazione completa [qui](https://firefox-source-docs.mozilla.org
...
+    _Su dispositivo mobile_: Firefox per iOS e Android invia per impostazione predefinita dati sulle campagne pubblicitarie mobili ad Adjust, il nostro fornitore di analytics, che ha una sua [informativa sulla privacy](https://www.adjust.com/terms/privacy-policy/).  I dati sulle campagne pubblicitarie mobili includono: ID pubblicitario Google o ID Android, indirizzo IP, indicatore temporale, Paese, lingua/impostazioni locali, sistema operativo e versione dell'app. Per saperne di più, consulta la [documentazione](https://dictionary.telemetry.mozilla.org/apps/fenix?itemType=metrics&page=1&search=adjust).
```
Comment (flodolo) [inline it/firefox_privacy_notice.md:89]: Italian doesn't use double space after a period.

File: zh-CN/firefox_privacy_notice.md [hunk 4]
```diff
@@ -26,7 +26,7 @@
 
 * __位置数据__：首次使用 Firefox 时，Firefox 会使用 IP 地址根据您所在的国家/地区设置默认搜索提供商。[了解更多信息](https://support.mozilla.org/kb/change-your-default-search-settings-firefox)。
 
-* __搜索查询__：默认情况下，如果您选择的搜索提供商支持搜索建议，Firefox 会将搜索查询发送给您的搜索提供商，以帮助您发现其他用户搜索的常用短语并改善您的搜索体验。[了解详情](https://support.mozilla.org/kb/search-suggestions-firefox)，包括如何禁用此功能。如果启用 Firefox Suggest，我们和我们的合作伙伴也可能会收到您的搜索查询。[在下方了解更多信息](#searches)。
+* __搜索查询__：默认情况下，如果您选择的搜索提供商支持搜索建议，Firefox 会将搜索查询发送给您的搜索提供商，以帮助您发现其他用户搜索的常用短语并改善您的搜索体验。[了解详情](https://support.mozilla.org/kb/search-suggestions-firefox)，包括如何禁用此功能。我们的默认搜索提供商的链接如下：
```
Comment (flodolo) [inline zh-CN/firefox_privacy_notice.md:29]: This translation is incomplete.

Where are the links and the sentence after the link??

**Review by flodolo — CHANGES_REQUESTED** [review]
There are problems with at least Italian and Chinese. I could fix Italian, but I can't do anything for Chinese.

File: it/firefox_privacy_notice.md (hunk 1 above)
Comment (CorneliuRusu) [reply it/firefox_privacy_notice.md]: Fixed.

File: it/firefox_privacy_notice.md (hunk 2 above)
Comment (CorneliuRusu) [reply it/firefox_privacy_notice.md]: Fixed.

File: zh-CN/firefox_privacy_notice.md (hunk 4 above)
Comment (CorneliuRusu) [reply zh-CN/firefox_privacy_notice.md:29]: Fixed

File: it/firefox_privacy_notice.md (hunk 3 above)
Comment (CorneliuRusu) [reply it/firefox_privacy_notice.md]: Fixed

**Comment by peiying2** [comment]
There are a few extra white spaces after a period in the paragraph that got updated.

File: cs/firefox_privacy_notice.md [hunk 5]
```diff
@@ -81,7 +86,7 @@ Dle výchozího nastavení u verzí Firefoxu pro stolní počítač vás požád
...
 
-    _U mobilních telefonů_: Firefox dle výchozího nastavení zasílá mobilní kampaně společnosti Adjust, našemu dodavateli analýz, který má své vlastní [zásady ochrany osobních údajů](https://www.adjust.com/terms/privacy-policy/). Údaje o mobilních kampaních zahrnují ID reklamy Google, adresu IP, časovou značku, zemi, jazyk/nastavení místního prostředí, operační systém a verzi aplikace. Přečtěte si [dokumentaci](https://dictionary.telemetry.mozilla.org/apps/fenix?itemType=metrics&page=1&search=adjust).
+    _U mobilních telefonů_: Firefox dle výchozího nastavení zasílá mobilní kampaně společnosti Adjust, našemu dodavateli analýz, který má své vlastní [zásady ochrany osobních údajů](https://www.adjust.com/terms/privacy-policy/).  Údaje o mobilních kampaních zahrnují ID reklamy Google nebo ID systému Android, adresu IP, časovou značku, zemi, jazyk/nastavení místního prostředí, operační systém a verzi aplikace. Přečtěte si [dokumentaci](https://dictionary.telemetry.mozilla.org/apps/fenix?itemType=metrics&page=1&search=adjust).
```
Comment (peiying2) [inline cs/firefox_privacy_notice.md:89]: extra space before `Údaje o …`

**Comment by flodolo** [comment]
I'm going to fix this myself, for sake of urgency.

One other thing that I noticed is that this pull request is coming from the main branch of your fork, while it should come from a branch. I've spent a lot of time fixing it and helping updating HTP instructions to avoid this, because it's very likely going to lead to issues (conflicts, unrelated changes, etc.).

**Comment by flodolo** [comment]
Apparently there a lot of double spaces after periods in translations, which doesn't make a lot of sense. I'll fix these anyway, but I have to open a different PR, since I don't have write access to the fork.

**Comment by BP-RazvanR-LE** [comment]
Hi Francesco,
We've been using a fork for a few years now and there were no problems. Pull requests were merged properly and we allowed editing for contributors. Functionally forking and branching are similar but with different intents.
There are more than one HTP engineer working on this project and we have different styles of dealing with it. For example, I prefer to have one single commit containing all languages, which I do from Git Bash.
My other colleague prefers to work exclusively in GitHub.

**Comment by BP-RazvanR-LE** [comment]
Thank you for your help with this.

**Comment by flodolo** [comment]
> We've been using a fork for a few years now and there were no problems. Pull requests were merged properly and we allowed editing for contributors. Functionally forking and branching are similar but with different intents.

Sorry but this is not accurate.

You don't have access to this repository, so you cannot create a branch directly here and you need to work on a fork.
When you work on a fork, you should never commit to the default branch (`main` in this case), because keeping the fork in sync will become a problem from that moment on. That's unless you plant to delete your fork every time, and recreate it from scratch.

This is an example of the problems that this creates (and created multiple times in the past)
https://github.com/mozilla/legal-docs/pull/1666 

It's not about having one commit or multiple commits either, or what you use to work on the repository (git is git, in the end, doesn't matter the front-end).

As for access, I tried to push and I got an "access denied" error, which I shouldn't normally get if the "allow edits from maintainers" is left checked.
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork

**Comment by BP-RazvanR-LE** [comment]
Keeping a fork in sync is automated on my computer. It's a matter of running three commands:
cd <to my local repository>
git fetch upstream
git merge upstream/main
And that's it.
Each time I start working on this type a project I run the script to sync my fork.

**Comment by CorneliuRusu** [comment]
Keeping a fork in sync is automated on my computer. It's a matter of running three commands:
cd "to my local repo"
git fetch upstream
git merge upstream/main
And that's it.
Every time I start working on this type a project I run the script to sync my fork.

**Comment by flodolo** [comment]
> git merge upstream/main

This will create a merge commit, because your history and the history of this repo diverged for the main branch. In the best scenario the merge commit is clean, but the example above shows that's not always the case. That extra history will show up in the pull request and confuse things unnecessarily.

It's easy to see if I'm wrong, just run those commands now and let me know if they run without issues.

The only way to avoid that if either committing to a branch, and keep main clean and in sync, or add a `git reset --hard upstream/main` instead of a `git merge`, then `git push -f` because you changed the history and need to force it.

**Pull request closed without merging**